packages/tocco-ui-showcase/src/toccoUtilExamples/notifier/example.js

```diff
@@ -38,24 +38,29 @@ class Example extends React.Component {
   confirmQuestion = () => {
     this.store.dispatch(notifier.confirm(
       'Title',
-      'Message Line 1<br/>Message Line 2',
-      'OK Text',
-      'Cancel Text',
-      () => console.log('ok pressed'),
-      () => console.log('cancel pressed')
+      'Message Line',
+      'OK text',
+      'Cancel text',
+      () => console.log('Ok was pressed'),
+      () => console.log('Cancel was pressed')
     ))
   }
 
   yesNoQuestion = () => {
     this.store.dispatch(notifier.yesNoQuestion(
-      'Title',
-      'Message Line 1<br/>Message Line 2',
-      'Yes Text',
-      'No Text',
-      'Cancel Text',
-      () => console.log('yes pressed'),
-      () => console.log('no pressed'),
-      () => console.log('Cancel pressed')
+      <Typography.H5>Title</Typography.H5>,
```

Comment:
Does it really make sense to pass a react node here? Shouldn't a title like the same all the time, hence only string (or resource id ) is allowed?

Reply:
TitleMessage.js handles four types of inputs (resource id, plain text, text containing html and nodes) and process props.title and props.message for all kind of toasters. To ensure identical presentation it would make sense to force resource id and wrap text automatically in certain tags (e.g. <h4> and <p>). I believe this is to tight and can not cope reality. By allowing all four types for all kind of toasters it is flexibel and comprehensible. May example yesNoQuestion should be improved to convey the purpose better (e.g. <Typography.H4>Title <Typography.Small>Byline</Typography.Small></Typography.H4>).